website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,12 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Added new currencies and countries
+Starting 2024-09-05, the Zimbabwean Gold (ZWG) and Zambian kwacha (ZMW) currencies
```

Comment:
```suggestion
Starting 2024-09-05, the Zimbabwe Gold (ZWG) and Zambian kwacha (ZMW) currencies
```

* https://en.wikipedia.org/wiki/Zimbabwean_ZiG
* AFAICT, most sites capitalize the first letter in Kwacha. :man_shrugging:

:man_shrugging: